common/fw-update-helper.cpp

```diff
@@ -191,7 +191,8 @@ namespace rs2
         // if device is D457, and fw is signed - using mipi specific procedure
         if (_is_signed
                 && (!strcmp(_dev.get_info(RS2_CAMERA_INFO_PRODUCT_ID), "ABCD")
-                 || !strcmp(_dev.get_info(RS2_CAMERA_INFO_PRODUCT_ID), "BBCD"))
+                 || !strcmp(_dev.get_info(RS2_CAMERA_INFO_PRODUCT_ID), "BBCD")
+                 || !strcmp(_dev.get_info(RS2_CAMERA_INFO_PRODUCT_ID), "ABCE"))
```

Comment:
Will we use the same PID for D437 recovery as D457 recovery?

Reply:
thanks, will look into if it's necessary to have separate recovery pid.